src/settings/OpenExceptionalForm/ServicePointSelector.js

```diff
@@ -29,7 +30,7 @@ class ServicePointSelector extends React.Component {
     const itemFormatter = (item) => (
       <li data-test-service-point key={item.id}>
         <div
-          className="CircleDiv"
+          className="circle-div"
```

Comment:
I checked your branch localy, and I think we can just remove this className. Then you can leave "style" as is, or wright it at the same line with the div. Since there is only one property here, both approaches are acceptable.

Reply:
I'm not sure why this got left as `circle-div`, it should have been made into `circleDiv`  but it does seem to be needed in order to make this div the proper height and circular dot.